web/init/src/components/kustomize/HelmValuesEditor.jsx

```diff
@@ -138,6 +142,18 @@ export default class HelmValuesEditor extends React.Component {
     }
   }
 
+  handleKeyboardSave = (e) => {
+    if (e.keyCode == 83 && (e.ctrlKey || e.metaKey)) {
```

Comment:
Would prefer if you name the keycode with a descriptive variable name as a constant

Reply:
Also `===`